src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -124,61 +129,95 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
 
     def tearDown(self):
+        """Tear down after each test case."""
         super(TestWithoutServers, self).tearDown()
 
         if self.fault_file:
             os.remove(self.fault_file)
 
+
 class TestWithServers(TestWithoutServers):
-    '''
-    Run tests with DAOS servers
+    """Run tests with DAOS servers and at least one client.
+
+    Optionally run DAOS clients on specified hosts.  By default run a single
+    DAOS client on the host executing the test.
```

Comment:
FWIW, (and not relevant to this patch, just an FYI), we don't really even want to run a daos client on the node running the test.

Ideally the node running the test should not be running the code being tested, so as to isolate any bad behaviour in the code being tested from the test running.

Reply:
That makes complete sense.  Obviously this comment should change if ever the AgentUtils behavior changes.